conduct.md

```diff
@@ -132,7 +132,7 @@ TODO: Write an introductory paragraph.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are the punishments handed out to those who break the rules or don't follow class conduct one or more times. The more that the class conduct is broken by the individual, the more severe the penalty is. Infractions are split upon different categories and each infraction category adds to a point total when committed. For example, if you commit three Level 1 infractions, that is just the same as committing 1 Level 2 infraction. The higher the total infraction points you have, the worse the punishment is.
```

Comment:
1- Please break this up to multiple lines.
2- I don't know if the word `punishment` is the most fitting here. Maybe consider replacing it with penalty. You'll also need to clarify what the penalty is in this case. Is it grade deduction or something else. Please elaborate more on that.